app/uk/gov/hmrc/tariffclassificationfrontend/controllers/CancelRulingController.scala

```diff
@@ -44,20 +46,19 @@ class CancelRulingController @Inject()(authenticatedAction: AuthenticatedAction,
 
   override protected def isValidCase(c: Case)(implicit request: AuthenticatedRequest[_]): Boolean = c.status == COMPLETED
 
+  private def cancelRuling(f: Form[CancelReason], caseRef: String)
```

Comment:
This code was repeated twice. 
I think now the code is more readable and maintainable.